web/setup.sh

```diff
@@ -0,0 +1,2 @@
+(cd web/html/src; yarn install)
+(cd web/html/src; yarn build)
```

Comment:
SuseGitExtraGenerationBuilder expect a filename returned as last line when calling setup.sh.

Not sure if you create a tar-ball with yarn like in `susemanager-nodejs-sdk-devel` or not.
If not, it makes sense to have at least an empty echo to prevent copying sources.

```
echo ""
```
might be sufficient.

Reply:
yes, I am just using the SuseGitExtraGenerationBuilder for runnings those scripts before packaging, so using echo "" works and it's more safe, we don't need to generate anything new in this case. 

Do you think it would be better to add this to the "build-packages-for-obs.sh" instead? I like the idea of tito having the control of it, as it's a pre-build dependency

Reply:
Hmm, then we would have two places where we call things like this. I think we should have just one place.

Reply:
So if you agree I will just keep it like that, already added the 'echo ""'